packages/grid/x-data-grid-pro/src/tests/editComponents.DataGridPro.test.tsx

```diff
@@ -589,6 +589,26 @@ describe('<DataGridPro /> - Edit Components', () => {
       expect(onValueChange.lastCall.args[1]).to.equal('Adidas');
     });
 
+    it('should call onCellEditStop', async () => {
+      const onCellEditStop = spy();
+
+      render(
+        <div>
+          <TestCase
+            onCellEditStop={onCellEditStop}
+          />
+          <div id="outside-grid"></div>
+        </div>
+      );
+
+      const cell = getCell(0, 0);
+      fireEvent.doubleClick(cell);
+      fireEvent.click(document.getElementById('outside-grid')!);
+      await act(() => Promise.resolve());
+
+      expect(onCellEditStop.callCount).to.equal(1);
+    });
+
```

Comment:
I've added a test, the same case when run manually in the browser works, but the test fails. I'm not sure if I should be adding some sort of delay after the outside click. Suggestions welcome.

Here is the test case: https://codesandbox.io/s/cool-yalow-c4d7x0?file=/demo.tsx

Reply:
Thanks a lot for the help :) The test is now passing.